docker-compose.yaml

```diff
@@ -25,6 +26,7 @@ services:
       - DOCKER_HOST=unix:///var/run/docker.sock
       - MAIN_CONTAINER_NAME='biomage-inframock-localstack'
       - LAMBDA_EXECUTOR=local
+      - EXTRA_CORS_ALLOWED_HEADERS=amz-sdk-request, amz-sdk-invocation-id
```

Comment:
And this line to add CORS header for these requests?

Reply:
yes